lib/utilities.js

```diff
@@ -7,7 +7,9 @@ export const importComponent = async (path) => {
   } catch (error) {
     let description = error.toString()
     let codeFrame = error.codeFrame
-    error.stack = [description, codeFrame].join("\n")
+
+    const location = `Location: ${path}`
+    error.stack = [location, description, codeFrame].join("\n")
```

Comment:
Thanks for improving this! I don’t know why I never though to log the file path here 😅 

Two things. I think “file” is probably a slightly clearer name. And I think we can lop off the beginning of the path and make it relative. So instead of `File: /Users/brandon/Code/charge/docs/pages.html.mdx` it would be `File: docs/pages.html.mdx`. That should make the file path stand out a bit more.

```suggestion
    const relativePath = path.replace(`${process.cwd()}${separator}`, "")
    const file = `File: ${relativePath}`
    error.stack = [file, description, codeFrame].join("\n")
```

This “path” import at the top would also need to be changed but I can’t seem to add a comment there to make the patch suggestion.

```diff
- import { normalize } from "path"
+ import { normalize, sep as separator } from "path"
```